### PR TITLE
180570785_change_price_format

### PR DESCRIPTION
**Pull request by radek-polcode**
#### What's this PR do?
- Uses comma as a delimiterfor thousands + display only two decimal places
- 
#### How should this be manually tested?
- Go to `localhost:3000/stake_accounts` and check how sols in first columns are displayed (they should be delimited by thousands and have 2 decimal places)

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/180570785

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

### CODE REVIEW

**Review by maciejkocylapc — CHANGES_REQUESTED** [review]
There is no delimiter for thousands

File: app/models/stake_account.rb [hunk 1]
```diff
@@ -37,4 +37,11 @@ class StakeAccount < ApplicationRecord
   scope :filter_by_account, ->(account) { where('stake_pubkey LIKE ?', "#{account}%") }
   scope :filter_by_staker, ->(staker) { where('staker LIKE ?', "#{staker}%") }
   scope :filter_by_withdrawer, ->(withdrawer) { where('withdrawer LIKE ?', "#{withdrawer}%") }
+
+  def formatted_delegated_stake
+    return "0.00" unless delegated_stake.is_a?(Numeric)
+
+    stake_in_sol = (delegated_stake / 1_000_000_000.to_f)
```
Comment (kbiala) [inline app/models/stake_account.rb:44]: Generally ok, but I'd make it a helper as well, it may be useful in case we want to transform lamports to sols in some other place.

File: test/helpers/application_helper_test.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class ApplicationHelperTest < ActiveSupport::TestCase
+  include ApplicationHelper
+
+  test "#lamports_as_formatted_sol returns formatted sol with delimiters" do
+    assert_equal "12,341.23", lamports_as_formatted_sol(12341232212414)
```
Comment (maciejkocylapc) [inline test/helpers/application_helper_test.rb:9]: You could also test for other formats like nil or string

**Review by maciejkocylapc — APPROVED** [review]
